src/arche/report.py

```diff
@@ -19,51 +19,63 @@ def wipe(self):
     def save(self, result):
         self.results[result.name] = result
 
-    def write_color_text(self, text, color=Fore.RED, style=""):
+    @staticmethod
+    def write_color_text(text, color=Fore.RED, style=""):
         print(color + style + text + Style.RESET_ALL)
 
-    def write_rule_name(self, rule_name):
+    @staticmethod
```

Comment:
Maybe all there static `write_something` methods could go into some sort of `utils` module.

Reply:
They belong to reporting, be it a module or a class with one simple goal - report something to jupyter output.